core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/benchmark/Main.java

```diff
@@ -24,7 +24,7 @@
 	}
 
 	public static void main(String[] args) throws RunnerException {
-		Options opt = new OptionsBuilder().include("")
+		Options opt = new OptionsBuilder().include("TargetBenchmarkInitialData.*")
 
 				// .addProfiler("stack", "lines=20;period=1;top=20")
 				.build();
```

Comment:
Use this main method to compare performance between SPARQL targets and predicate+object targets. The benchmarks test how the number of targets already present in the database affect the performance of a standard number (and size) of transactions.